lib/magma.gd

```diff
@@ -483,18 +483,21 @@ DeclareAttribute( "TrivialSubmagmaWithOne", IsMagmaWithOne );
 
 #############################################################################
 ##
-#P  IsAssociative( <M> )  . . . . . . . . test whether a magma is associative
+#P  IsAssociative( <M> )  . .  test whether a mult. elt. coll. is associative
 ##
 ##  <#GAPDoc Label="IsAssociative">
 ##  <ManSection>
 ##  <Prop Name="IsAssociative" Arg='M'/>
 ##
 ##  <Description>
-##  A magma <A>M</A> is <E>associative</E> if for all elements
+##  A collection <A>M</A> of multiplicative elements
+##  is <E>associative</E> if for all elements
 ##  <M>a, b, c \in</M> <A>M</A> the equality
 ##  <M>(a</M><C> * </C><M>b)</M><C> * </C><M>c =
 ##  a</M><C> * </C><M>(b</M><C> * </C><M>c)</M> holds.
 ##  <P/>
+##  A collection of multiplicative elements that is closed under the
+##  multiplication is called a <E>magma</E>.
```

Comment:
I understand. Do you have a suggestion for how I could improve this? Otherwise I might just remove this new sentence - the intention was to be helpful, but now it is potentially confusing.

The real 'problem' here is that this documentation previously only applied to magmas, which is why it was (and still is) in the chapter on magmas, and why magmas didn't need to be defined in this manual entry.

Now that `IsAssociative` applies more generally, perhaps this documentation should instead appear in a different file, and perhaps in a different chapter - what do you think?